### DISCUSSION TITLE
Very minor inconsistency: "Homing" menu item misspelled as "Homming"

### DISCUSSION THREAD

**Discussion by GKSeifert** [mriscoc/Ender3V2S1#447] [General]
This is not a problem, but a minor blemish. In Ender3V2-422-MM-20221002 I see "Z after Homming" at the bottom of the Homing menu. I don't recall it being that way in the previous release. If this is not the appropriate place to bring up this kind of issue I apologize, please let me know where it belongs.

**Comment by cvail55** [comment]
Yes because we wanted the nozzle to raise to a specified amount after homing.  So there is an option to raise it to your desired level or have it sit on the bed. There is no issue and the option is where it's supposed to be.

**Reply by GKSeifert** [reply]
Yep, I have been using it for quite a while.  But there should only be one 'm' in homing. I did say it was minor, and only a blemish!

**Reply by mriscoc** [reply]
Thanks.

**Comment by cvail55** [comment]
HAHAHA I didnt even catch that xD Nice one!!
